<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="sys_ui_module">
    <sys_ui_module action="INSERT_OR_UPDATE">
        <active>true</active>
        <application display_value="Sample Exception">a709fe70db1a7700f15d5c7a489619af</application>
        <filter/>
        <name>Sample Exceptions</name>
        <order/>
        <path/>
        <path_relative_to_root>false</path_relative_to_root>
        <roles>x_354377_exception.sample_exception_user</roles>
        <sys_class_name>sys_ui_module</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2019-06-25 08:32:32</sys_created_on>
        <sys_domain>global</sys_domain>
        <sys_domain_path>/</sys_domain_path>
        <sys_id>6b09fe70db1a7700f15d5c7a489619b1</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>Sample Exceptions</sys_name>
        <sys_package display_value="Exception" source="x_354377_exception">baa83630db1a7700f15d5c7a48961994</sys_package>
        <sys_policy/>
        <sys_scope display_value="Exception">baa83630db1a7700f15d5c7a48961994</sys_scope>
        <sys_update_name>sys_ui_module_6b09fe70db1a7700f15d5c7a489619b1</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2019-06-25 08:32:32</sys_updated_on>
        <table>x_354377_exception_sample_exception</table>
        <uncancelable>false</uncancelable>
        <view_name/>
    </sys_ui_module>
</record_update>
